jsonobject/jsonobject.go

```diff
@@ -1,4 +1,4 @@
-package dl
+package jsonobject
```

Comment:
Would it be better if it is `json`?
Other packages are like `client`, `crypto`, `ledger,`  and `rpc`, so `jsonobject` looks not consistent a bit to me.

More importantly, I thought this JSON file is intended for general JSON serialization and deserialization, but it's doing something for `nonce`, which is contract argument specific.
I think we should separate them. (maybe, withNonce should be done outside of the file?)

Reply:
@feeblefakie 

Thank you for your feedback.

> Would it be better if it is json?
> Other packages are like client, crypto, ledger, and rpc, so jsonobject looks not consistent a bit to me.

Understood.
I revise the package name to `json` in https://github.com/scalar-labs/scalardl-go-client-sdk/pull/18/commits/74c5845765ab88dc1f7939ba6bc1388a26da428a

and, according to Go linter's recommendation.
I renamed the structure `JSONObject` to `Object`.
(Go linter consider that `json.JSONObject` stutters and it recommend just `json.Object`)

> More importantly, I thought this JSON file is intended for general JSON serialization and deserialization, but it's doing something for nonce, which is contract argument specific.
I think we should separate them. (maybe, withNonce should be done outside of the file?)

I see. I removed the WithNonce function in https://github.com/scalar-labs/scalardl-go-client-sdk/pull/18/commits/f6c8bd34e604714ecce50122f427317914b379a5
This part will be handled later in the `client/service` implementation directly.

Please take a look again.
Thank you!